x-pack/platform/test/functional_with_es_ssl/apps/triggers_actions_ui/stack_alerts_page.ts

```diff
@@ -33,6 +33,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       shouldUseHashForSubUrl: false,
     });
 
+  // eslint-disable-next-line ban/ban
```

Comment:
Why was this added?

Reply:
Hmm it's definitely removed on `main` so probably a bad merge, yeah